distribution/docs/pom.xml

```diff
@@ -47,6 +47,9 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <properties>
+        <ddf-ui.version>5.0.8-SNAPSHOT</ddf-ui.version>
```

Comment:
Should we plan on getting out a new release of DDF-UI?

Reply:
Potentially. it makes sense to me